lib/Rest/GeneralRequest.cpp

```diff
@@ -198,7 +194,7 @@ std::string const& GeneralRequest::header(std::string const& key,
   auto it = _headers.find(key);
   if (it == _headers.end()) {
     found = false;
-    return StaticStrings::Empty;
```

Comment:
maybe better to return std::string_view?

Reply:
I prefer not to do it as part of this PR. There are lots of callers that do `std::string const& value = _request->header(name, found);` which would all need to be changed.
I am not against it in general, but it would make the scope of the PR larger. This is intended to be only a small change.